app/models/credentials.rb

```diff
@@ -15,13 +15,6 @@ class Credentials < Sequel::Model
   # Bcrypt work factor, minimum recommended work factor is 12
   BCRYPT_COST = 12
 
-  # special characters according to https://www.owasp.org/index.php/Password_special_characters
-  VALID_PASSWORD_REGEX = %r{^(?=.*?[A-Z].*[A-Z])                             # 2 uppercase letters
-                             (?=.*?[a-z].*[a-z])                             # 2 lowercase letters
-                             (?=.*?[0-9])                                    # 1 digit
-                             (?=.*[ !"#$%&'()*+,-./:;<=>?@\[\\\]^_`{|}~]).  # 1 special character
-                             {12,128}$}x.freeze                                     # 12-128 characters
-
```

Comment:
can you please change only this in this file?

Reply:
Done